src/components/content/home/ServiceDashboardByIsvSkeleton.tsx

```diff
@@ -0,0 +1,15 @@
+/*
```

Comment:
we make a new folder under home - `dashboard` and under that two more folders -> `user` and `isv` and move classes under them